app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    @classmethod
+    def couple_create(cls,
```

Comment:
fix that was said earlier) or text me in direct messages in Slack if u dont uderstand my advice)